packages/dev/docs/pages/releases/2023-04-04.mdx

```diff
@@ -0,0 +1,96 @@
+* Genericize `useFocus()` for narrowed event handler types - [@imhoffd](https://github.com/imhoffd) - [PR](https://github.com/adobe/react-spectrum/pull/4095)
+
+## Fixes
+
+* Fix SSR issues with overlays and mismatching tabIndex - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4297)
+* Fix clearing date picker by setting value to null - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4312)
+* Fix additional character appended to Calendar cells in various locales - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/4170)
+* Fix overlay position clamping and arrow position - [@LFDanLu](https://github.com/LFDanLu) and [@filipw01](https://github.com/filipw01) - [PR](https://github.com/adobe/react-spectrum/pull/4220)
+* Stop propagation when escape key is pressed in TooltipTrigger to prevent dialog from closing - [@audreyso](https://github.com/audreyso) - [PR](https://github.com/adobe/react-spectrum/pull/4203)
+* NumberField: screen reader fails to announce negative values with currencySign: 'accounting' - [@majornista](https://github.com/majornista) - [PR](https://github.com/adobe/react-spectrum/pull/4161)
+* Fix rapidly changing values in useSpinButton causing live region to announce values that are no longer relevant - [@majornista](https://github.com/majornista) - [PR](https://github.com/adobe/react-spectrum/pull/4159)
+* ListBox: avoid overriding Item id with custom id - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/4179)
+* Remove label prop from Form that is unused - [@majornista](https://github.com/majornista) - [PR](https://github.com/adobe/react-spectrum/pull/4209)
+* Fix duplicate id in some icons - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/4217)
+* TS Strict update react-aria 'b' and 'L' - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/3930)
+* Specify box sizing where we rely on it - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/4180)
+* Add negative margin on VisuallyHidden on both sides - [@filipw01](https://github.com/filipw01) - [PR](https://github.com/adobe/react-spectrum/pull/4177)
+* Generate table row textValue from row header cells by default - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4290)
+* Fix combobox visible focus styling when opening the menu via keyboard - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/4299)
+* Fix font issues with CCJK and synthesized italics - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/4238)
+
```

Comment:
we just merged https://github.com/adobe/react-spectrum/pull/4164 as well

Reply:
yep, need to update with all the things that have been merged since I made this.